app/Http/Controllers/Api/InventoryController1.php

```diff
@@ -0,0 +1,51 @@
+<?php namespace App\Http\Controllers;
+
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+use App\Item;
+
+use Illuminate\Support\Facades\Response;
+use Illuminate\Support\Facades\Input;
+
+class InventoryController1 extends Controller {
```

Comment:
We name the controller classes for each team number to avoid conflict. We will merge them before integrate testing.